django_any/compat.py

```diff
@@ -45,3 +47,10 @@ def get_remote_field_model(field):
         return field.remote_field.model
     else:
         return field.rel.to
+
+
+if django.VERSION < (2, 0):
+    CommaSeparatedIntegerField = models.CommaSeparatedIntegerField
+else:
+    class CommaSeparatedIntegerField(models.CharField):
```

Comment:
What is the reason to support the field dropped from django? What is the difference between compat.CommaSeparatedIntegerField and any other CharField with validator defined?

Reply:
CommaSeparatedIntegerField is not dropped in 1.8. There is no difference, but if we want to drop this field, we need to remove it from tests or add many version checks.

Reply:
Do you think we should register handler for CommaSeparatedIntegerField for django >= 1.11 here: https://github.com/coagulant/django-whatever/blob/master/django_any/models.py#L139?

Reply:
I saved CommaSeparatedIntegerField-related code only for compatibility with 1.8 and my goal was "do as few changes as possible". After April 2018 (or even now if you want) we can simply remove all this code and all my placeholders.